rllib/algorithms/algorithm_config.py

```diff
@@ -2384,6 +2389,11 @@ def offline_data(
         output_config=NotProvided,
         output_compress_columns=NotProvided,
         output_max_file_size=NotProvided,
+        output_data_write_method=NotProvided,
```

Comment:
Btw, this is not the fault of this PR, but can we clean up the missing typehints in `AlgorithmConfig.offline_data()`? Not sure why these weren't here in the first place.

e.g.:
```
output_data_write_method: Optional[some type ..] = NotProvided,
```